data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3360 @@
+			choice
+				`	"I will see what I can do."`
+					accept
+				`	"I have no interested in bringing you weapons, even the least deadly."`
+					decline
+	
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog `You have not yet brought any kinds of non lethal weapons to Ya Hi, look harder!`
+	to complete
+		or
+			has "Bring Pug Staffs: done"
+			has "Bring Coalition Weapons: done"
+			has "Bring Void Rifle: done"
+	on complete
+		clear "Unfettered: offered close quarter"
+		log `Brought very strong stunning weapons to the Unfettered so they can do their training without risk, or at least that is what they said, apparently there may be more to it.`
+		conversation
+			`You go see Hi Ye with the non lethal rifles to see if he has anything else to ask you. He seems very happy you managed to find them, and says, "Great job! It will be decisive in the coming war."`
+			choice
+				`	"How can a few better trained troops make a difference?"`
+				`	"Glad I could make a difference then."`
+					goto getrd
+			`	"Well... To be completely honest with you these rifles are not only intended for training purposes but I cannot tell you any more for now."`
+			
+			label getrd
+			`	"We will soon want your help with the upcoming invasion, get ready for it! Having a strong flagship is always an important asset but it is not all." It is not clear whether he means more ships or better strategy, and knowing him it could be either.`
+			choice
+				`	"The Basilisk will not be enough I take it?"`
+				`	"Okay in that case I will get ready."`
+					goto train
+			`	"The Basilisk is very strong, but it is only a prototype, it has a few weaknesses."`
+			
+			label train
+			`	"Remember, having a strong fleet and flagship does not matter much if you do not know how to use it properly! If you want to learn how to do that there are a lot of Unfettered captains looking forward to challenging you on the job board."`
+
+mission "Bring Pug Staffs"
+	invisible
+	landing
+	source "Warfeed"
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Pug Peacekeeping Staff" 10
+		dialog `Do you want to sell 10 Pug Peacekeeping Staff to the Unfettered? Hi Ye says they can give you <payment> for it.`
+	on accept
+		payment 2500000
+		outfit "Pug Peacekeeping Staff" -10
+		dialog `The Unfettered thank you for bringing these 10 staffs to them. They give you <payment>.`
+
+mission "Bring Coalition Weapons"
+	invisible
+	landing
+	source "Warfeed"
+	to offer
+		has "Unfettered: Close Quarter: offered"
+	on offer
+		require "Enforcer Riot Staff" 10
+		dialog `Do you want to sell 10 Enforcer Riot Staffs to the Unfettered? Hi Ye says they can give you <payment> for it.`
+	on accept
+		payment 2000000
```

Comment:
I would suggest flipping the payments on these, as Heliarch h2h weapons are actually much more difficult to get than Pug.

Also it may be good to make these repeatable, but at a higher quantity, say 50 each or so, to demonstrate supplying them for larger scale training.

Reply:
You're right on the first point. Right now at least.
However as part of the intro of the Heliarch you get access to them I think. They're still further away though.

But it isn't for large scale training at all, he doesn't want to lose too much money on it.
It's to board the Wanderers without killing them but with a lot of firepower.

Reply:
If I make it a repeatable it'll be way too easy to farm money with this

Reply:
I'll add a little something saying "oh what are these? gimme more" when you have between 1 and 49 of them then